unit-tests/test_autocomplete.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python3
+
+"""Tests for keeper interactive shell autocompletion."""
+
+from unittest import TestCase
+
+# This crufty import makes autocomplete import successfully.
+import keepercommander.cli
+
+import keepercommander.autocomplete as autocomplete
+
+# And this silences the IDE/Editor warning from importing something we don't intend to use.
+_ = keepercommander.cli
+
+
+def check_roundtrip_w_in_double_quotes(string):
+    """Test string for escape -> unescape, both with double quotes."""
+    escaped = autocomplete.escape_string(True, string)
+    unescaped = autocomplete.unescape_string(True, escaped)
+    assert string == unescaped
+
+
+def check_roundtrip_wo_in_double_quotes(string):
+    """Test string for escape -> unescape, both without double quotes."""
+    escaped = autocomplete.escape_string(False, string)
+    unescaped = autocomplete.unescape_string(False, escaped)
+    assert string == unescaped
+
+
+class TestAutocompleteQuotingWithInDoubleQuotes(TestCase):
```

Comment:
Please use parameterized tests to simplify the following tests. See https://github.com/Keeper-Security/Commander/blob/master/unit-tests/commands/helpers/test_timeout.py

Reply:
Done.